docs/node_setup.md

```diff
@@ -1,127 +1,125 @@
+# Install Rust and Substrate
+$ curl https://getsubstrate.io -sSf | bash -s -- --fast
 
-cargo build --release
+# Build the binary
+$ cd HydraDX-node/
+$ cargo build --release
 ```
 
-##### Running binary
-```bash
-./target/release/hydra-dx --chain lerna --name your-validator-name --validator
+If you built the binary following the steps above, the path to your binary is:
+```
+HydraDX-node/target/release/hydra-dx
 ```
 
-:::important
-`--name` will be dislayed in [Telemetry](https://telemetry.polkadot.io/#list/HydraDX%20Snakenet). Telemetry show all nodes in HydraDX network. It is important to use unique name so you can identify your node.
-:::
+### 04 Run the binary
+You can run the binary by executing the following command:
+
+```bash
+$ {PATH_TO_YOUR_BINARY} --chain lerna --name {YOUR_NODE_NAME} --validator
+```
 
-### Running validator node
+Besides the path to your binary (see above), you need to specify a node name which will be used to identify your node in [Telemetry](https://telemetry.polkadot.io/#list/HydraDX%20Snakenet) where you can find a list of all nodes running on HydraDX Snakenet.
 
-#### Systemd
-You can run your validator as a systemd process so that it will automatically restart on server reboots or crashes.
+### 05 Running with systemd
+To make sure that your node is automatically started when your machine reboots, we recommend running the HydraDX node as a systemd process. To do so, create the following file and insert the content while replacing the variables indicated as `{VARIABLE}`:
 
-First create file `hydradx-validator.service` in `/etc/systemd/system/`
 ```bash
-touch /etc/systemd/system/hydradx-validator.service
+$ touch /etc/systemd/system/hydradx-validator.service
 ```
 
-whith content:
-:::important
-replace `{...}` with correct values e.g `User={YOUR_SYSTEM_USER}` -> `User=Hydra`
-:::
-
 ```
 [Unit]
 Description=HydraDX validator
 
 [Service]
 Type=exec
 User={YOUR_SYSTEM_USER}
-ExecStart={PATH_TO_YOUR_BINARY} --chain lerna --name {YOUR_TELEMETRY_NAME} --validator
+ExecStart={PATH_TO_YOUR_BINARY} --chain lerna --name {YOUR_NODE_NAME} --validator
 Restart=always
 RestartSec=120
 
 [Install]
 WantedBy=multi-user.target
 ```
-:::caution
-It's recommended to delay the restart of a node with `RestartSec` in the case of node crashes. It's possible that when a node crashes, consensus votes in GRANDPA aren't persisted to disk. In this case, there is potential to equivocate when immediately restarting. What can happen is the node will not recognize votes that didn't make it to disk, and will then cast conflicting votes. Delaying the restart will allow the network to progress past potentially conflicting votes, at which point other nodes will not accept them.
+
+:::note
+Setting RestartSec is recommended because it delays the restart of the node in the case of a crash. This allows for any unpersisted data (such as consensus votes) to be written to the disk before the node is restarted.
```

Comment:
We should include some info about why. (equivocation = double signing / committing the same data twice which will result in a slash)